atlas-2024-odfr/mkjson.py

```diff
@@ -0,0 +1,257 @@
+    }
+
+
+# Populate fields
+
+# This is applicable for the pp data only!
+evergreen_data = {
+    # Accelerator - just CERN LHC
+    "accelerator": "CERN-LHC",
+    # ATLAS Collaboration; recid only if we need a specific author list
+    "collaboration": {
+      "name": "ATLAS collaboration",
+     },
+    # Basic collision data - this applies only to the pp data
+    "collision_information": {
+      "energy": "13TeV",
+      "type": "pp"
+    },
+    # Published this year!
+    "date_published": "2024",
+    # ATLAS experiment
+    "experiment": [
+      "ATLAS"
+    ],
+    # Thanks to the Open Data Portal
+    "publisher": "CERN Open Data Portal",
+    # Note: beginning of the reprocessing
+    "date_reprocessed": "2020",
+    "distribution": {
+      "formats": [
+        "DAOD_PHYSLITE",
+        "root"
+      ],
+    },
+    # Dataset type information for Open Data Portal
+    "type": {
+      "primary": "Dataset",
+    },
+    # Information about usage
+    "usage": {
+      "description": "<p> The data and MC simulation provided by the ATLAS experiment in DAOD_PHYSLITE format is released under a CC BY license. This format can be used directly like a ROOT ntuple (or using uproot) for simple studies or processed into secondary ntuples with systematic uncertainties included using the ATLAS AnalysisBase software. <p>Extensive instructions for interacting with the data, as well as documentation of the dataset naming conventions and their contents, are provided on the ATLAS Open Data website linked below.",
+      "links": [
+        {
+          "description": "ATLAS Open Data Website",
+          "url": "http://opendata.atlas.cern"
+        },
+        {
+          "description": "Resources to understand and use the open data for research",
+          "url": "https://opendata.atlas.cern/docs/userpath/researchers"
+        },
+        {
+          "description": "ATLAS Analysis Software Tutorial",
+          "url": "https://atlassoftwaredocs.web.cern.ch/ASWTutorial/TutorialWeek/"
+        }
+      ]
+    },
+    # Information about (production) methodology
+    'methodology': {
+      'description':'<p>These data were created during LS2 as part of a major reprocessing campaign of the Run 2 data. All data were reprocessed using Athena Release 22, and new corresponding MC simulation samples were produced, in an MC simulation campaign called MC20a. These data and MC simulation datasets were processed into DAOD_PHSYLITE format files; this is a light-weight data format intended for general analysis use, sufficient to support a wide variety of ATLAS analyses.'},
+    "license": {
+      "attribution": "These data are provided under the <a href='https://creativecommons.org/licenses/by/4.0/'>CC BY</a> license. The citation policy can be <a href='https://opendata.atlas.cern/docs/documentation/ethical_legal/citation_policy'>found here</a>."
```

Comment:
We try to standardise the "attribution" field value to use the [SPDX](https://spdx.org/licenses/) license code names. These have controlled values and it's good to use them for possible interoperability, discoverability on aggregated servers, etc.

Hence the idea would be to store only a brief code name for the license in question (CC0-1.0, CC-BY-4.0, etc).
The accompanying text to display is currently added by the application logic (living in Jinja templates) that takes the value and displays appropriate "enhanced" text around the license in standardised way for all records having that license.  (So that individual records won't have to repeat this.)

Therefore please store only the brief license code in `license.attribution` field and we can add the accompanying text in the application logic.

P.S. That said, I would advise to use the CC0 license as per the [CERN open data policy for the LHC experiments](https://cds.cern.ch/record/2745133/files/CERN-OPEN-2020-013.pdf).

Reply:
Thanks @tiborsimko ! I've updated to CC0-1.0.

For future posterity, I had a nice chat with Sunje Dallmeier-Tiessen about this. CC BY requires the assertion of a copyright claim in order to require attribution (otherwise one can't enforce a demand for a citation). The data itself cannot be copyrighted (one cannot claim a copyright over a "fact"). Formatted data (e.g. a table) can be copyrighted, so it seems as though our data could (likely) be copyrighted, since it is heavily processed and formatted, but this is not an area that has been litigated significantly in many jurisdictions. The safer approach is to simply not claim a copyright (so that no one can accuse us of falsely asserting copyright), and to rely on the professional ethics and standard practices of the field to "nicely request" a citation (as we do now).